## Frame Parameters
# ROBOT_FRAME: "base_link"

# Robot Size Parameters
USE_FOOTPRINT: true
FOOTPRINT:
  [[-0.2800, -0.1600], [-0.2800, 0.1600], [0.1200, 0.1600], [0.1200, -0.1600]]
ROBOT_RADIUS: 0.1 # [m], If param "USE_FOOTPRINT" is set to true, this param is not used
FOOTPRINT_PADDING: 0.01 # [m], If localmap contains padding, you should set this param to 0.0

# Robot Configuration Parameters
MAX_LINEAR_VELOCITY: 0.45 # [m/s]
MIN_LINEAR_VELOCITY: -0.2 # [m/s]
MAX_ANGULAR_VELOCITY: 0.5 # [rad/s]
MIN_ANGULAR_VELOCITY: 0.1 # [rad/s]
# MAX_IN_PLACE_YAWRATE: 0.6 # [rad/s]
# MIN_IN_PLACE_YAWRATE: 0.3 # [rad/s]
MAX_LINEAR_ACCELERATION: 0.5 # [m/s^2]
MAX_LINEAR_DECELERATION: 2.0 # [m/s^2]
MAX_ANGULAR_ACCELERATION: 3.2 # [rad/s^2]
